data/officialLanguages.yml

```diff
@@ -285,7 +285,8 @@ PK: [ur, en, pa]
 PL: [pl]
 PM: [fr]
 PN: [en, pih]
-PR: [en]
+# Spanish is the predominant spoken language and the only language on signs, but English is co-official
```

Comment:
```suggestion
```
No need for an explanation for something that would not be controversial. (After all, it is this way in the Wikipeda too)